Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM debian:unstable
```

Comment:
I think it would be better to use the official php image

Reply:
I'd rather do that as soon as I get the current solution to work...! (check the TODO)